tests/api-mongo.test.js

```diff
@@ -466,6 +466,39 @@ describe('MongoDB Words', () => {
         });
     });
 
+    it('should return a list of igbo terms when using english by using single qoutes', (done) => {
```

Comment:
you have a typo, it should be `quotes`, instead of `qoutes`

Reply:
@Effiti Looks like addressing this typo is the last thing!

Reply:
I am sorry.. english is not my main language.. sorry